2.10.3 build failure on <= OS X 10.11 with <= Xcode 8.2.1

```
[ 50%] Building CXX object src/CMakeFiles/cpprest.dir/uri/uri_builder.cpp.o
cd /tmp/cpprestsdk-20180806-90982-cgtnjd/cpprestsdk-2.10.3/src && /usr/local/Homebrew/Library/Homebrew/shims/mac/super/clang++  -DCPPREST_FORCE_HTTP_CLIENT_ASIO -DCPPREST_FORCE_HTTP_LISTENER_ASIO -Dcpprest_EXPORTS -I/tmp/cpprestsdk-20180806-90982-cgtnjd/cpprestsdk-2.10.3/Release/include -I/tmp/cpprestsdk-20180806-90982-cgtnjd/cpprestsdk-2.10.3/Release/src/pch -I/tmp/cpprestsdk-20180806-90982-cgtnjd/cpprestsdk-2.10.3/Release/libs/websocketpp -I/usr/local/include -isystem /usr/local/opt/openssl/include  -stdlib=libc++ -Wno-return-type-c-linkage -Wno-unneeded-internal-declaration -std=c++11 -fno-strict-aliasing -DNDEBUG -fPIC   -Werror -pedantic -Wall -Wextra -Wcast-qual -Wconversion -Wformat=2 -Winit-self -Winvalid-pch -Wmissing-format-attribute -Wmissing-include-dirs -Wpacked -Wredundant-decls -Wno-overloaded-virtual -Wno-sign-conversion -Wno-deprecated -Wno-unknown-pragmas -Wno-reorder -Wno-char-subscripts -Wno-switch -Wno-unused-parameter -Wno-unused-variable -Wno-unused-value -Wno-unknown-warning-option -Wno-return-type-c-linkage -Wno-unused-function -Wno-sign-compare -Wno-shorten-64-to-32 -Wno-unused-local-typedefs -Wno-delete-non-virtual-dtor -Wno-unused-lambda-capture -o CMakeFiles/cpprest.dir/uri/uri_builder.cpp.o -c /tmp/cpprestsdk-20180806-90982-cgtnjd/cpprestsdk-2.10.3/Release/src/uri/uri_builder.cpp
[ 53%] Building CXX object src/CMakeFiles/cpprest.dir/utilities/asyncrt_utils.cpp.o
cd /tmp/cpprestsdk-20180806-90982-cgtnjd/cpprestsdk-2.10.3/src && /usr/local/Homebrew/Library/Homebrew/shims/mac/super/clang++  -DCPPREST_FORCE_HTTP_CLIENT_ASIO -DCPPREST_FORCE_HTTP_LISTENER_ASIO -Dcpprest_EXPORTS -I/tmp/cpprestsdk-20180806-90982-cgtnjd/cpprestsdk-2.10.3/Release/include -I/tmp/cpprestsdk-20180806-90982-cgtnjd/cpprestsdk-2.10.3/Release/src/pch -I/tmp/cpprestsdk-20180806-90982-cgtnjd/cpprestsdk-2.10.3/Release/libs/websocketpp -I/usr/local/include -isystem /usr/local/opt/openssl/include  -stdlib=libc++ -Wno-return-type-c-linkage -Wno-unneeded-internal-declaration -std=c++11 -fno-strict-aliasing -DNDEBUG -fPIC   -Werror -pedantic -Wall -Wextra -Wcast-qual -Wconversion -Wformat=2 -Winit-self -Winvalid-pch -Wmissing-format-attribute -Wmissing-include-dirs -Wpacked -Wredundant-decls -Wno-overloaded-virtual -Wno-sign-conversion -Wno-deprecated -Wno-unknown-pragmas -Wno-reorder -Wno-char-subscripts -Wno-switch -Wno-unused-parameter -Wno-unused-variable -Wno-unused-value -Wno-unknown-warning-option -Wno-return-type-c-linkage -Wno-unused-function -Wno-sign-compare -Wno-shorten-64-to-32 -Wno-unused-local-typedefs -Wno-delete-non-virtual-dtor -Wno-unused-lambda-capture -o CMakeFiles/cpprest.dir/utilities/asyncrt_utils.cpp.o -c /tmp/cpprestsdk-20180806-90982-cgtnjd/cpprestsdk-2.10.3/Release/src/utilities/asyncrt_utils.cpp
/tmp/cpprestsdk-20180806-90982-cgtnjd/cpprestsdk-2.10.3/Release/src/utilities/asyncrt_utils.cpp:54:32: error: default initialization of an object of const type 'const (anonymous namespace)::to_lower_ch_impl' without a user-provided default constructor
    constexpr to_lower_ch_impl to_lower_ch;
                               ^
                                          {}
/tmp/cpprestsdk-20180806-90982-cgtnjd/cpprestsdk-2.10.3/Release/src/utilities/asyncrt_utils.cpp:65:32: error: default initialization of an object of const type 'const (anonymous namespace)::eq_lower_ch_impl' without a user-provided default constructor
    constexpr eq_lower_ch_impl eq_lower_ch;
                               ^
                                          {}
/tmp/cpprestsdk-20180806-90982-cgtnjd/cpprestsdk-2.10.3/Release/src/utilities/asyncrt_utils.cpp:76:32: error: default initialization of an object of const type 'const (anonymous namespace)::lt_lower_ch_impl' without a user-provided default constructor
    constexpr lt_lower_ch_impl lt_lower_ch;
                               ^
                                          {}
3 errors generated.
make[2]: *** [src/CMakeFiles/cpprest.dir/utilities/asyncrt_utils.cpp.o] Error 1
make[1]: *** [src/CMakeFiles/cpprest.dir/all] Error 2
make: *** [all] Error 2
```
https://gist.github.com/ilovezfs/a2407f4b21b0eca4549850acc03556d2
https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/28272/version=el_capitan/console

--- a/Release/src/utilities/asyncrt_utils.cpp
+++ b/Release/src/utilities/asyncrt_utils.cpp
@@ -51,7 +51,7 @@ namespace
         }
     };
 
-    constexpr to_lower_ch_impl to_lower_ch;
+    constexpr to_lower_ch_impl to_lower_ch{};
 
     struct eq_lower_ch_impl
     {
@@ -62,7 +62,7 @@ namespace
         }
     };
 
-    constexpr eq_lower_ch_impl eq_lower_ch;
+    constexpr eq_lower_ch_impl eq_lower_ch{};
 
     struct lt_lower_ch_impl
     {
@@ -73,7 +73,7 @@ namespace
         }
     };
 
-    constexpr lt_lower_ch_impl lt_lower_ch;
+    constexpr lt_lower_ch_impl lt_lower_ch{};
 }
 
 namespace utility

